### DISCUSSION TITLE
Enable recording of drop columns in refactor log to avoid having to disable blockondataloss

### DISCUSSION THREAD

**Discussion by simonsabin** [microsoft/DacFx#219] [Ideas]
If one wants to drop a column you have to disable BlockOnPossibleDataLoss, this is a blanket flag and something folks don't want to leave disabled. 

If the dropping of a column was recorded somewhere, then the deployment could ignore the loss of that data as its been explicitly requested to be dropped.

**Comment by ErikEJ** [comment]
@simonsabin I usually do destructive things like that in a pre pre-deployment script (seperate pipeline stage)

**Reply by simonsabin** [reply]
Agree, and thats one option, it would be nice to not have to do that.

**Reply by ErikEJ** [reply]
I never use refactorlog... and prefer being explicit.

**Reply by simonsabin** [reply]
I don't think many do. because you've got to do things in a particular way to do it
